pkgs/development/python-modules/minichain/default.nix

```
@@ -0,0 +1,65 @@
+{ buildPythonPackage
+, eliot
+, fetchPypi
+, google-search-results
+, jinja2
+, lib
+, manifest-ml
+, openai
+, pytestCheckHook
+, pythonRelaxDepsHook
+}:
+
+buildPythonPackage rec {
+  pname = "minichain";
+  version = "0.3.3";
```

```suggestion
  version = "0.3.3";
  format = "setuptools";
```
It seems to be preferable to set `format = "setuptools";` explicitly these days.